Solutions/GoogleCloudPlatformNAT/Data Connectors/GCPNATLogs_ccp/GCPNATLogs_DCR.json

```diff
@@ -0,0 +1,127 @@
labels']['region']), RouterId=tostring(Resource['labels']['router_id']), ProjectId=tostring(Resource['labels']['project_id']), PayloadType=tostring(ProtoPayload['@type']), PrincipalEmail=tostring(ProtoPayload['authenticationInfo']['principalEmail']), PrincipalSubject=tostring(ProtoPayload['authenticationInfo']['principalSubject']), CallerIp=tostring(ProtoPayload['requestMetadata']['callerIp']), UserAgent=tostring(ProtoPayload['requestMetadata']['callerSuppliedUserAgent']), RequestAttributeTime=todatetime(ProtoPayload['requestMetadata']['requestAttributes']['time']), RequestAttributeAuth=tostring(ProtoPayload['requestMetadata']['requestAttributes']['auth']), RequestAttributeDestination=tostring(ProtoPayload['requestMetadata']['destinationAttributes']), PayloadRequestNats=tostring(ProtoPayload['request']['nats']), RequestType=tostring(ProtoPayload['request']['@type']), RequestName=tostring(ProtoPayload['request']['name']), RequestNetwork=tostring(ProtoPayload['request']['network']), EncryptedInterconnectRouter=tobool(ProtoPayload['request']['encryptedInterconnectRouter']), RequestRegion=tostring(ProtoPayload['request']['region']), RequestSelfLink=tostring(ProtoPayload['request']['selfLink']), RequestId=tostring(ProtoPayload['request']['id']), ServiceName=tostring(ProtoPayload['serviceName']), MethodName=tostring(ProtoPayload['methodName']), AuthorizationInfo=tostring(ProtoPayload['authorizationInfo']), ResourceName=tostring(ProtoPayload['resourceName']), ResponseType=tostring(ProtoPayload['response']['@type']), ResponseName=tostring(ProtoPayload['response']['name']), ResponseUser=tostring(ProtoPayload['response']['user']), ResponseTargetId=tostring(ProtoPayload['response']['targetId']), ResponseStatus=tostring(ProtoPayload['response']['status']), ResponseProgress=tostring(ProtoPayload['response']['progress']), ResponseStartTime=todatetime(ProtoPayload['response']['startTime']), ResponseRegion=tostring(ProtoPayload['response']['region']), ResponseSelfLinkWithId=tostring(ProtoPayload['response']['selfLinkWithId']), ResponseInsertTime=todatetime(ProtoPayload['response']['insertTime']), ResponseTargetLink=tostring(ProtoPayload['response']['targetLink']), ResponseSelfLink=tostring(ProtoPayload['response']['selfLink']), ResponseId=tostring(ProtoPayload['response']['id']), ResponseOperationType=tostring(ProtoPayload['response']['operationType']), ResponseErrors=tostring(ProtoPayload['response']['errors']), ResponseErrorCode=toint(ProtoPayload['response']['error']['code']), ResponseErrorMessage=tostring(ProtoPayload['response']['error']['message']), ResourceLocation=tostring(ProtoPayload['resourceLocation']), OperationId=tostring(Operation['id']), OperationProducer=tostring(Operation['producer']), OperationFirst=tobool(Operation['first']), OperationLast=tobool(Operation['last']) | where ResourceType == 'gce_router' and ServiceName == 'compute.googleapis.com' and MethodName has 'v1.compute.routers.' | project-away protoPayload, resource, labels, operation",
```

Comment:
Don't use project-away, use project in smart manner, so that standard table creation will go smoothly